package.json

```diff
@@ -14,12 +14,12 @@
   "scripts": {
     "test": "babel-node ./node_modules/jasmine/bin/jasmine.js",
     "prebuild": "rimraf dist",
-    "build": "NODE_ENV=production npm-run-all --parallel build:*",
-    "build:main": "babel --copy-files --out-dir dist src",
+    "build": "tsc && NODE_ENV=production npm-run-all --parallel build:*",
+    "build:main": "babel --copy-files --out-dir dist dist",
     "build:umd": "webpack --mode=production --output-filename roundware.umd.js",
-    "build:umd.min": "webpack -p --output-filename roundware.umd.min.js",
+    "build:umd.min": "webpack --mode production --output-filename roundware.umd.min.js",
```

Comment:
Overall looks pretty good, I've just got a few high-level comments. We will want to build TypeScript with webpack+babel instead of using `tsc` directly here, as you suggest. @hburgund In tandem, this would be a good time to fix the issue where we can't test with a local build of the repo.

Reply:
This sounds like the right approach, though I'm new to `tsc`. Would be fantastic to fix that weird local build testing issue...